src/domTree.js

```diff
@@ -125,7 +125,26 @@ const toMarkup = function(tagName: string): string {
     return markup;
 };
 
-export type CssStyle = {[name: string]: string};
+export type CssStyle = {
```

Comment:
Please change this to an exact object type, e.g.
```
export type CssStyle = {|
    backgroundColor?: string,
    ...
|}
```
This will cause `flow` to complain about any misspelled properties.